config.toml

```diff
@@ -66,6 +66,7 @@ time_format_blog = "Monday, January 02, 2006"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
+future = "v1.18"
```

Comment:
@jimangel , nit: is there a `development` or `devbranch` param (instead of future)?
That way there is a connection to the actual branch name?

Reply:
We can absolutely use a different name other than `future`, however, I don't think this param will be exclusively used for dev / release branches. I could see it coming in handy to make a page more dynamic down the road like: https://kubernetes.io/docs/setup/release/version-skew-policy/ (clearly out of date and `kubectl` being an example of one that needs a future version)

Reply:
If you want to be very future-proof, call it something like `next-minor-version` (and allow room for a major-version future release).

Reply:
Good. Future [what?] seems too generic.